files/sinonjs/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "npm",
+  "name": "sinon",
+  "repo": "sinonjs/sinon",
+  "files": {
+    "basePath": "lib/",
+    "include": ["sinon.js"]
```

Comment:
I dont think that file is ready to work in the browser. This is what we need http://sinonjs.org/releases/sinon-1.17.3.js